components/director/internal/runtimemapping/runtime_verifier.go

```diff
@@ -0,0 +1,50 @@
+package runtimemapping
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/kyma-incubator/compass/components/director/internal/labelfilter"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/pkg/errors"
+)
+
+const (
+	consoleURLLabelKey = "runtime/console_url"
+	dexSubdomain       = "dex"
+	consoleSubdomain   = "console"
+)
+
+//go:generate mockery -name=RuntimeRepository -output=automock -outpkg=automock -case=underscore
+
+type RuntimeRepository interface {
+	GetByFiltersGlobal(ctx context.Context, filter []*labelfilter.LabelFilter) (*model.Runtime, error)
+}
+
+type runtimeVerifier struct {
+	logger      Logger
+	runtimeRepo RuntimeRepository
+}
+
+func NewRuntimeVerifier(logger Logger, runtimeRepo RuntimeRepository) *runtimeVerifier {
```

Comment:
why this is not part of runtime service? If you would like to keep this struct, can you explain to me why it is called Verifier when if just get runtime by some Label?

Reply:
Actually, very good idea.